x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/defense_evasion_execution_lolbas_wuauclt.json

```diff
@@ -48,5 +48,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 4
+  "version": 5
```

Comment:
As above -- is this an unnecessary version bump? Or was it missed in the previous batch of 8.2 updates?